ibc/host/keys.go

```diff
@@ -0,0 +1,285 @@
+package host
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Store key prefixes for IBC
+const (
+	KeyClientStorePrefix       = "clients"
```

Comment:
confirming that these need to be visibile outside of this package. Lowercase if not

Reply:
Yeah these will be used by all the other ICS components when interfacing with the stores - need to be exposed